costmodel/cluster.go

```diff
@@ -142,7 +142,7 @@ func resultToTotal(qr interface{}) (map[string][][]string, error) {
 }
 
 // ClusterCostsForAllClusters gives the cluster costs averaged over a window of time for all clusters.
-func ClusterCostsForAllClusters(cli prometheusClient.Client, cloud costAnalyzerCloud.Provider, windowString, offset string) (map[string]*Totals, error) {
+func ClusterCostsForAllClusters(cli prometheusClient.Client, windowString, offset string) (map[string]*Totals, error) {
```

Comment:
We actually do need this, see: https://github.com/kubecost/cost-model/pull/210/files#diff-58d9ebd2d60cdef59d19220180531c6fR151

Reply:
Ok I'll put that back